build.sbt

```diff
@@ -4,21 +4,23 @@ scalaModuleSettings
 
 name                       := "scala-partest"
 
-version                    := "1.0.11-SNAPSHOT"
+version                    := "1.0.12-SNAPSHOT"
 
 scalaVersion               := crossScalaVersions.value.head
 
 crossScalaVersions         := {
   val java = System.getProperty("java.version")
   if (java.startsWith("1.6."))
-    Seq("2.11.6", "2.12.0-M1")
+    Seq("2.11.7", "2.12.0-M1")
   else if (java.startsWith("1.8."))
-    Seq("2.12.0-M2")
+    Seq("2.12.0-M2", "2.12.0-M3")
   else
     sys.error(s"don't know what Scala versions to build on $java")
 }
 
-scalaXmlVersion            := "1.0.4"
+scalaXmlVersion            := {
+  if(scalaVersion.value == "2.12.0-M3") "1.0.5" else "1.0.4"
```

Comment:
Actually, the way it is at the moment is easier. We need 1.0.5 for 2.12.0-M3 and 1.0.4 for 2.12.0-M2, 2.12.0-M1 and 2.11.